src/parsers/power_system_table_data.jl

```diff
@@ -849,9 +849,14 @@ function make_reactive_params(gen)
     reactive_power = get(gen, :reactive_power, 0.0)
     reactive_power_limits_min = get(gen, :reactive_power_limits_min, nothing)
     reactive_power_limits_max = get(gen, :reactive_power_limits_max, nothing)
-    reactive_power_limits =
-        (isnothing(reactive_power_limits_min) & isnothing(reactive_power_limits_max)) ?
-        nothing : (min = reactive_power_limits_min, max = reactive_power_limits_max)
+    if (isnothing(reactive_power_limits_min) & isnothing(reactive_power_limits_max))
```

Comment:
```suggestion
    if isnothing(reactive_power_limits_min) && isnothing(reactive_power_limits_max)
```

Reply:
@claytonpbarrows as per the message I will commit the suggestion and merge